docs/txp.md

```diff
@@ -296,12 +301,25 @@ Stakes modification is a bit more complex, but is also very
 intuitive. We have `TxOut`s corresponding to inputs and outputs of
 transaction. Each `TxOut` can be converted to a list of
 `(StakeholderId, Coin)` pairs. It depends on `AddrStakeDistribution`
-of given address. For `BootstrapEraDistr` the value of `TxOut` is
-distributed among bootstrap stakeholders proportional to their weights.
-If distribution is `SingleKeyDistr id`, then the value of `TxOut` will
-be assigned to `id`. In case of `MultiKeyDistr` stake will be
-distributed among multiple stakeholders according to specified
-portions. These lists are concatenated and we have two lists of
+of given address.
+* For `BootstrapEraDistr` the value of `TxOut` is distributed among
+bootstrap stakeholders proportional to their weights. The behavior
+depends on whether the value is less than the sum of weights of all
+stakeholders (this sum is called `bootDustThreshold`). If it's greater
+than or equal to `bootDustThreshold`, then each stakeholder receives
+`val / weights_sum` coins (`val` is the value of `TxOut`) and one
+stakeholder also receives the remainder (the choice is
+deterministc). If it's less than `bootDustThreshold`, then some
+stakeholders will receive the same stake as their weights, one
+stakeholder may receive less then their weight and other stakeholders
+won't receive anything.
+* If distribution is `SingleKeyDistr id`, then the value of `TxOut` will
+be assigned to `id`.
+* In case of `MultiKeyDistr` stake will be distributed among multiple
+stakeholders according to specified portions. The first stakeholder
+may receive slightly more than others due to rounding.
+
+Then these lists are concatenated and we have two lists of
 `(StakeholderId, Coin)` pairs: the first one is how much stake each
```

Comment:
you wrote
>We have TxOuts corresponding to inputs and outputs of transaction. Each TxOut can be converted to a list of (StakeholderId, Coin) pairs. It depends on 

if so then it should be:
Then these lists are concatenated and we have two lists of (StakeholderId, Coin) pairs: the first one is how much stake each stakeholder should **lose**, the second one is how much stake each stakeholder should **gain**